aries/agents/mediator/tests/mediator-oob-invitation.rs

```diff
@@ -36,7 +36,7 @@ fn endpoint_register_returns_oob_with_correct_accept_header() -> Result<()> {
 
```

Comment:
We can do away with this test. Since we no longer have a dual purpose (html/json format) endpoint.